content/en/docs/setup/platform-setup/_index.md

```diff
@@ -7,6 +7,12 @@ aliases:
 keywords: [platform-setup]
 ---
 
-{{< tip >}}
-Istio {{< istio_version >}} has been tested with these Kubernetes releases: {{< supported_kubernetes_versions >}}.
-{{< /tip >}}
+We recommend using a version of Kubernetes that has been tested with Istio:
+
+|Istio Version       | Kubernetes Version
```

Comment:
I don't think this is a good place for this table because this is specifically the Istio 1.6 setup page, so users shouldn't need to look for it in a table of historical releases. All they need is the tip that says what versions of kube they need to be using with this release.

If you absolutely do not want to put it on the release landing page, as I suggested, then I prefer you keep it in the version-policy doc, where you had it, although I think we're more likely to forget to update it there when we add a new release. Putting it with all the release notes will make it easier to remember to add a new line at the same time when we add a new release.